torchvision/transforms/v2/_temporal.py

```diff
@@ -7,6 +7,19 @@
 
 
 class UniformTemporalSubsample(Transform):
+    """[BETA] Uniformly subsamples ``num_samples`` indices from the temporal dimension of the video.
+
+    .. betastatus:: UniformTemporalSubsample transform
+
+    Videos are expected to be of shape ``(..., T, C, H, W)`` where ``T`` is the temporal dimesion.
```

Comment:
I think we used square brackets everywhere else

```suggestion
    Videos are expected to be of shape ``[..., T, C, H, W]`` where ``T`` denotes the temporal dimension.
```